zhaquirks/tuya/ts0601_din_power.py

```diff
@@ -233,6 +241,197 @@ def __init__(self, *args, **kwargs):
     }
 
 
+class Tuya3PhaseElectricalMeasurement(ElectricalMeasurement, TuyaLocalCluster):
+    """Tuya Electrical Measurement cluster."""
+
+    _CONSTANT_ATTRIBUTES = {
+        ElectricalMeasurement.AttributeDefs.ac_current_divisor.id: 1000,
+        ElectricalMeasurement.AttributeDefs.ac_voltage_divisor.id: 10,
+        ElectricalMeasurement.AttributeDefs.ac_frequency_divisor.id: 100,
+    }
+
+
+(
+    TuyaQuirkBuilder("_TZE204_ugekduaj", "TS0601")
+    .tuya_dp(
+        dp_id=0x65,
+        ep_attribute=Tuya3PhaseElectricalMeasurement.ep_attribute,
+        attribute_name="ac_frequency",
+    )
+    # Energy consumed
+    .tuya_sensor(
+        dp_id=0x01,
```

Comment:
Ok, one last thing, can we keep all `dp_id`s as decimals?
That's how we have them everywhere else, so just to stay consistent then.